FlymFork/src/main/res/values/strings.xml

```diff
@@ -226,11 +226,11 @@
     <string name="fontsize_entrylist">Font size in articles'\ list</string>
     <string name="settings_volume_buttons_action">Action by volume buttons</string>
     <string name="behaviour">Behavior</string>
-    <string name="question_delete_entry">Delete article ?</string>
+    <string name="question_delete_entry">Delete this article?</string>
     <string name="share_all_text">Share whole text</string>
     <string name="group">Group</string>
     <string name="auto_image_load">Auto images load</string>
-    <string name="show_progress_info">Detailed progress info</string>
```

Comment:
Is PROGRESS technical detail info

Reply:
I propose "Detailed progress technical info".

Reply:
ok